include/curl/curl.h

```diff
@@ -2157,6 +2157,11 @@ typedef enum {
   /* websockets options */
   CURLOPT(CURLOPT_WS_OPTIONS, CURLOPTTYPE_LONG, 320),
 
+#ifdef MPTCP
+  /* Set Multipath TCP */
+  CURLOPT(CURLOPT_TCP_MULTIPATH, CURLOPTTYPE_LONG, 316),
+#endif
+
```

Comment:
This is a public header, it cannot use `#ifdefs` like this. The symbols need to be provided no matter what.